docker-compose.yaml

```diff
@@ -0,0 +1,11 @@
+services:
```

Comment:
even though I am glad you took the time to craft this, we don't want to support docker compose. Please drop this file.